packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -7,8 +7,58 @@ import { Canvas, Title } from '@storybook/addon-docs';
 
 If you are not using React, you can still embed a calling/chat composite into your html page with a simple `<script>` tag and several lines of code like:
 
+You can easily find prebuilt bundles from:
+
+[The latest bundles for chat composite](https://github.com/Azure/communication-ui-library/releases/latest/download/chatComposite.js)
+
+[The latest bundles for call composite](https://github.com/Azure/communication-ui-library/releases/latest/download/callComposite.js)
+
+For development and prototyping purpose, import these 2 urls with `<script>` tags will be the fastest way. In production environment, we recommend that serving composite files on your own CDN will help with performance.
+
+## Use Composites in html
+
+After importing chat/calling composite bundle, you are able to load composites using these 2 code snippets:
+
+```tsx
+  const chatAdapter =   The for composite which allows you control composite externally,
+    await chatComposite.loadChatComposite( // This function will be exposed to window.chatComposite once you import composite bundle
+      {
+        displayName, // string
+        threadId, // string
+        endpointUrl, // string
+        userId, // Object { communicationUserId: string }
+        token // string
+      },
+      elementToRender, // Container element
+      props // Object for all props(except adapter) of composite, optional, check composite page for more details
```

Comment:
I couldn't understand what 

> Object fo all props(except adapter)

means.

Reply:
Updated, this is an object contains all optional props for the adapter, like showTitle, showParticipant